model-optimizer/extensions/front/tf/pooling_ext.py

```diff
@@ -72,20 +72,44 @@ def extract(cls, node):
         return cls.enabled
 
 
-def create_pooling_attrs(node, pool_method):
+class AvgPoolV2FrontExtractor(FrontExtractorOp):
+    op = 'AvgPoolV2'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node):
+        attrs = create_pooling_attrs(node, 'avg', is_pooling_v2=True)
+        attrs.update({'op': __class__.op})
+        PoolingV2.update_node_stat(node, attrs)
+        return cls.enabled
+
+
+class MaxPoolV2FrontExtractor(FrontExtractorOp):
+    op = 'MaxPoolV2'
+    enabled = True
+
+    @classmethod
+    def extract(cls, node):
+        attrs = create_pooling_attrs(node, 'max', is_pooling_v2=True)
+        attrs.update({'op': __class__.op})
+        PoolingV2.update_node_stat(node, attrs)
+        return cls.enabled
+
+
+def create_pooling_attrs(node, pool_method, is_pooling_v2=False):
```

Comment:
No need to use parameter `is_pooling_v2`. Instead just check that the specific attribute exists in the node attrs like this: `if 'strides' in node.pb.attr`

Reply:
Agree, have rewritten with the use of `hasattr(..., 'strides')`. Needed to removed extractor unittests in order to pass successfully precomit